### PR TITLE
Allow uppercase chars in Shared Folder name

### PR DESCRIPTION
**Pull request by 61f7b7437262085c97876892ddf9c0d8**
The username validator is too restrictive, removed completely.

NethServer/dev#5247

### CODE REVIEW

**Comment by gsanchietti** [comment]
In-progress...

**Review by gsanchietti — CHANGES_REQUESTED** [review]
This PR is still in progress.

File: root/usr/share/nethesis/NethServer/Language/en/NethServer_Module_SharedFolder.php [hunk 1]
```diff
@@ -41,3 +41,4 @@
 $L['migrate-vhost_Message'] = 'This shared folder will be deleted and all data will be moved inside the virtual host. Migrate the shared folder to virtual host `${0}`?';
 $L['Migrate_label'] = 'Migrate';
 $L['__ANY__'] = 'default';
+$L['valid_platform,ibay-validator,ibay-name,3'] = 'The following characters are not allowed \ / : * ? " < > |';
```
Comment (DavidePrincipi) [inline root/usr/share/nethesis/NethServer/Language/en/NethServer_Module_SharedFolder.php]: The label key must be fixed: ``valid_platform,ibay-create,ibay-name,3`` (``validator`` becomes ``create``).

**Review by DavidePrincipi — CHANGES_REQUESTED** [review]

**Review by DavidePrincipi — APPROVED** [review]

File: createlinks [hunk 2]
```diff
@@ -56,6 +56,7 @@ event_actions('nethserver-ibays-update', qw(
 validator_actions('ibay-create', qw(
                   uniq-account 20
                   ibay-length 30
+				  ibay-name 20
```
Comment (gsanchietti) [inline createlinks]: Please correctly indent this line.

File: root/etc/e-smith/validators/actions/ibay-name [hunk 3]
```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/perl
+#
+# Copyright (C) 2012 Nethesis S.r.l.
```
Comment (gsanchietti) [inline root/etc/e-smith/validators/actions/ibay-name]: Change the year to 2017.

**Review by gsanchietti — APPROVED** [review]

**Pull request merged**